src/Client.cpp

```diff
@@ -153,50 +177,62 @@ int UrsaClient::start() {
         }
     }
 
-    // wait for status thread to terminate
-    status_th.join();
+    status_th.detach();
```

Comment:
It seems wrong to detach the thread

Reply:
idk why but when doing `join()` there is a random hang from time to time, not able to reproduce on `Debug` build :| do you have any clue?

Reply:
already resolved, there was a data race